docs/source/api_index.rst

```diff
@@ -410,13 +411,13 @@ Data Check Classes
     evalml.data_checks.NullDataCheck
     evalml.data_checks.IDColumnsDataCheck
     evalml.data_checks.TargetLeakageDataCheck
-    evalml.data_checks.OutliersDataCheck
     evalml.data_checks.NoVarianceDataCheck
     evalml.data_checks.ClassImbalanceDataCheck
     evalml.data_checks.MulticollinearityDataCheck
     evalml.data_checks.DateTimeFormatDataCheck
     evalml.data_checks.TimeSeriesParametersDataCheck
     evalml.data_checks.TimeSeriesSplittingDataCheck
+    evalml.data_checks.outliers_data_check.OutliersDataCheck
```

Comment:
Another example of https://github.com/alteryx/evalml/issues/2548

Reply:
(Failed build: https://readthedocs.com/projects/feature-labs-inc-evalml/builds/852750/)